### PR TITLE
Play Fair Cipher Algorithm in Cpp

### PR DESCRIPTION
**Pull request by ravireddy07**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

- Added code and updated README.md

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#6727 
(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/cryptography/PlayFair_Cipher.cpp [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    std::string key = "harry";
+    std::string plaintext = "my name is ravi";
+    std::string encryptedtext, decryptedtext;
+    /*
+    // Custom Input:
+    cout << "Enter The Key: ";
+    getline(cin, key);
+    cout << "\nEnter the Plain Text to Encrypt: ";
+    getline(cin, plaintext);
+    */
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cryptography/PlayFair_Cipher.cpp]: remove this part

File: C-Plus-Plus/cryptography/PlayFair_Cipher.cpp [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    return 0;
+}
+
+/*
+Sample Output:
+
+Plain Text is : my name is ravi
+Encrypted Text is : SFKBLFMOYRUK
+Decrypted Text is: MYNAMEISRAVI
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cryptography/PlayFair_Cipher.cpp]: write the output in the way your code is producing

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top

File: C-Plus-Plus/cryptography/PlayFair_Cipher.cpp (hunk 1 above)
Comment (ravireddy07) [reply C-Plus-Plus/cryptography/PlayFair_Cipher.cpp]: Done

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**